README.md

```diff
@@ -110,6 +110,25 @@ new Fingerprint2({
 });
 ```
 
+#### `preprocessor` option
+
+Since some components can be changed frequently, this option may help you improve hash accuracy. You can modify any component value before hash calculation. For example: strip browser version from user agent.
```

Comment:
This section should start with an explanation of what the `preprocessor` option does, something along the lines of "Function that is called with each component value that may be used to modify component values before computing the fingerprint". The user agent example is good.